test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -646,23 +665,12 @@ def test_cgroup_cpuset_and_memory(self):
         jid = self.server.submit(j)
         a = {'job_state': 'R'}
         self.server.expect(JOB, a, jid)
-        self.server.status(JOB, [ATTR_o, ATTR_e], jid)
```

Comment:
Restart mom explicitly before running this test

Reply:
Done.